.github/workflows/ci.yml

```diff
@@ -4,15 +4,13 @@ on: [push, workflow_dispatch]
 
 jobs:
   tests:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     strategy:
       matrix:
         python-version:
-        - "3.8"
-        - "3.9"
         - "3.10"
         - "3.11"
-        # - "3.12"
+        - "3.12"
```

Comment:
Should 3.13 be added as well?

Reply:
I tried - but 3.13 is broken due to touchstone related dependencies, see https://github.com/mitodl/hq/issues/6821